### ISSUE TITLE
fail2ban not working if ncdata on USBdrive

### ISSUE THREAD

**Issue by peter-sc** [nextcloud/nextcloudpi#324]
Hello, after the NC data is moved to USB drive, then the config `/etc/fail2ban/jail.conf`  should also change the location of `nextcloud.log` from `/var/www/nextcloud/data/nextcloud.log` to `/media/USBdrive/ncdata/nextcloud.log`.

From:
```
/etc/fail2ban/jail.conf 
[nextcloud]
enabled  = true
port     = http,https
filter   = nextcloud
logpath  = /var/www/nextcloud/data/nextcloud.log
maxretry = 3
```

To:
```
/etc/fail2ban/jail.conf 
[nextcloud]
enabled  = true
port     = http,https
filter   = nextcloud
logpath  = /media/USBdrive/ncdata/nextcloud.log
maxretry = 3
```

**Comment by nachoparker** [comment]
There's a  PR that fixes this

https://github.com/nextcloud/nextcloudpi/pull/323

thanks for reporting

**Comment by nachoparker** [comment]
fixed in `v0.48.1`
